VeroliBGV/BGV008.xml

```diff
@@ -0,0 +1,836 @@
+                           <item>
+                              <abbr>ወ</abbr> or <abbr>ወበ</abbr> for <expan>ወበእንተ፡</expan> (e.g. on <locus target="#169va #171rb"/>)
+                           </item>
+                           <item>
+                              <abbr>ተ</abbr> or <abbr>ተፈ</abbr> for <expan>ተፈሥሒ፡</expan> (e.g. on <locus target="#168ra #169vb"/>)
+                           </item>
+                           <item>
+                              <abbr>ሠ</abbr> or <abbr>ሠረ</abbr> for <expan>ሠረቀ፡</expan> (e.g. on <locus target="#158rb #158vb"/>)
+                           </item>
+                           <item>
+                              <abbr>ወ፡ ሣ፡ ይ፡</abbr> for <expan>ወልድኪ፡ ሣህሎ፡ ይክፍለነ፡</expan> (e.g. on <locus target="#173vb #174rb #175ra"/>)
+                           </item>
+                        </list>
+                     </handNote>
+                  </handDesc>
+                
+                  <additions>
+                     <list>                                      
+                        <item xml:id="e1">
+                           <locus target="1r"/>
+                           <desc>The shelfmark "8 Zanutto" is written in pencil, with reference to <bibl>
+                              <ptr target="bm:Zanutto1932Bibliografia"/>
+                              <citedRange unit="page">92</citedRange>
+                           </bibl>.</desc>
+                        </item>
+                        <item xml:id="e2">
+                           <desc>Frequent erasures (e.g. <locus target="#3v #51v #106r"/>), corrections over erasure or interlineally (e.g. <locus target="#11v #45v #58r"/>) throughout 
+                              the entire manuscript.
+                           </desc>
+                        </item>
+                        <item xml:id="e3">
+                           <desc>A <foreign xml:lang="gez">Tamallas</foreign>-signs in the shape of a cross are written on <locus target="#43r #161ra"/>  to indicate the omission of a 
+                              line of text or a word. The omitted portion of text is written in the upper margin. It is encircled in black dots on <locus target="#161ra"/>.
+                           </desc>
+                        </item>
+                        </list> 
+                  </additions>
+            
+                  <bindingDesc>
+                     <binding contemporary="true">
+                        <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards.
+                        </decoNote>
+                        <decoNote xml:id="b2" type="bindingMaterial">
+                           <material key="wood"/>  
+                        </decoNote>
+                        <decoNote xml:id="b3" type="SewingStations">4</decoNote>
+                        <decoNote xml:id="b4" type="Spine">Two pairs of chains of the endband stitches are visibile at the top and the bottom of the spine cover.</decoNote>
```

Comment:
"head" and "tail" instead of "top" and "bottom"?

Reply:
Make sense

Reply:
I will leave this to @marcodibella , who I'm sure will provide a consistent and appropriate description of this features. My description of the binding is to be considered provisional.